docs/products/alloydbomni/reference/list-of-extensions.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Extensions for Aiven for AlloyDB Omni
+sidebar_label: Extensions
+---
+
+Aiven for AlloyDB Omni extensions enhance capabilities of your service by adding advanced
+functionalities such as performance optimizations or additional data types.
+
+## List supported extensions
+
+To list the extensions and their details, such as extension version numbers,
+connect to your service and run:
+
+```sql
+SELECT *
+FROM pg_available_extensions;
+```
+
+## Limitations
```

Comment:
This doesn't apply to Google's Alloydb-omni, as we give the alloydbadmin role to our users which allow them to create extensions normally restricted to superusers.